### ISSUE TITLE
Server Dialog Box

### ISSUE THREAD

**Issue by swarfega** [roboirc/kvirc#6]
Once you have selected the server(s) you want to join, the server choose dialog should automatically disappear.

**Comment by roboirc** [comment]
What if you want to open multiple connections to different servers? Need the server dialog box still.

**Comment by roboirc** [comment]
I think it would be better to have the huge server list on right click popup menu for new irc context / window.

**Comment by swarfega** [comment]
If you need to select another server after closing the dialog box, you can use the connect to icon on the tool bar which provides a quick shortcut.

**Comment by roboirc** [comment]
Yes but Connect To only provides 10 recent connections. What if you want something not in list and only in Servers and Configuration?
